vllm/model_executor/layers/sampler.py

```diff
@@ -177,8 +178,7 @@ def _should_modify_greedy_probs_inplace(self) -> bool:
         This is used by speculative decoding, which requires that the sampling
         method be encoded into the probability distribution.
         """
-        # Modify greedy probs if include_gpu_probs_tensor is set.
-        return self.include_gpu_probs_tensor
+        return self.should_modify_greedy_probs_inplace
```

Comment:
Should just make `should_modify_greedy_probs_inplace` as the property?

Reply:
Good idea, will be addressed by https://github.com/vllm-project/vllm/pull/6971